transport/TransportImpl.cpp

```diff
@@ -1076,24 +1076,12 @@ void TransportImpl::internalIceReceived(Endpoint& endpoint,
     auto dataReceiver = _dataReceiver.load();
     if (_rtpIceSession->isAttached(&endpoint))
     {
-        if (ice::isResponse(packet->get()) && _rtpIceSession->isResponseAuthentic(packet->get(), packet->getLength()))
-        {
-        }
-        else if (ice::isRequest(packet->get()) &&
-            _rtpIceSession->isRequestAuthentic(packet->get(), packet->getLength()))
-        {
-        }
-        else
+        if (!_rtpIceSession->isIceAuthentic(packet->get(), packet->getLength()))
         {
             // TODO attack, add metric for this
             return;
         }
 
-        if (_rtpIceSession->canAcceptNewRemoteCandidate(timestamp, source))
-        {
-            endpoint.registerListener(source, this);
-        }
-
```

Comment:
We accept can accept candidates any time. The IceSession is going to call "onIceAccpted" and it will be registered on that moment